File: Single.java
class Animal
{
    int a=10,b=20;
    void print()
    {
        System.out.println("Animals are good.");
    }
}
class dog extends Animal
{
    void display()
    {
        System.out.println("Dog is also an  animal.");
    }
}
class Single
{
    public static void main(String args[])
    {
    dog obj = new dog();
    obj.display();
    obj.print();
    System.out.println(obj.a + " "+ obj.b);
    }
    
}

File: Single.cpp
#include<iostream>
using namespace std;
class animal
{
    public:
    int a = 10, b= 20;
    public:
    void print()
    {
        cout << "Animals are good."<< endl;
    }
};
class dog : public animal
{
    public:
    void display()
    {
        cout << "dog is also is onr amimal" << endl; 
    }
};
int main()
{
    dog obj;
    obj.print();
    obj.display();
    cout << obj.a << " " << obj.b << endl;
    return 0;
}